bee-node/src/tools/password.rs

```diff
@@ -23,11 +25,17 @@ pub enum PasswordError {
 pub struct PasswordTool {}
 
 pub fn exec(_tool: &PasswordTool) -> Result<(), PasswordError> {
-    let password = read_password_from_tty(Some("Password: "))?;
-    let password_reenter = read_password_from_tty(Some("Re-enter password: "))?;
-    if password != password_reenter {
-        return Err(PasswordError::NonMatching);
+    let password;
+    if let Ok(env_password) = env::var("BEE_TOOL_PASSWORD") {
+        password = env_password;
```

Comment:
Just make the if/else return the password instead

Reply:
I also wanted to comment on that. I'ld suggest the following (Thibault probably means the same):
```rust
let password = if let Ok(env_password) = .... {
     env_password
} else {
   ...
   password
}
```